src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
Do we have/need an app context?

There are other ways to solve the disposable problem as well. Making everything disposable removes options and is misleading